english/33hours/explore-room/check-toilet/check-toilet.md

```diff
@@ -0,0 +1,3 @@
+Entered into the toilet,the noise was as a result of the driping of water from the tap on a sachet of a used soap sachet.
```

Comment:
It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.